### DISCUSSION TITLE
QeRaman

### DISCUSSION THREAD

**Discussion by Santoshkumarkaushik** [nguyen-group/QERaman#5] [Q&A] [answered]
Which tag can be used in assume_isolated in scf calculation for bilayer hetero structure of 2d tmds?

**Comment by nguyen-group** [comment, marked as answer]
Hi, 
For the bilayer hetero TMD, you can do the same way for the MoS2 monolayer (with k-points: x  x  1).
If you mean assume_isolated = '2D' in scf.in, this tag is used to correct the soft phonon in the 2D system. It can also be used for the bilayer case.
